ifupdown2/addons/vxlan.py

```diff
@@ -47,7 +47,7 @@ class vxlan(Addon, moduleBase):
             },
             "vxlan-local-tunnelip": {
                 "help": "vxlan local tunnel ip",
-                "validvals": ["<ipv4>"],
+                "validvals": ["<ipv4>, <ipv6>"],
                 "example": ["vxlan-local-tunnelip 172.16.20.103"]
             },
```

Comment:
We shouldn't add ipv6 to this attribute but add a new attribute `vxlan-local-tunnelip6`